### PR TITLE
 app-crypt/zuluCrypt: bump version to 5.4.0

### PR DESCRIPTION
**Pull request by soredake**
Addressing package.mask entry:
>Doesn't build with glibc-2.25

Tested with 2.26-r3, build's fine.

>doesn't build

Build's now.

>needs a maintainer

Here i am.

### CODE REVIEW

**Comment by jonasstein** [comment]
> The name section should contain only lowercase non-accented letters, the digits 0-9, hyphens, underscores and plus characters. Uppercase characters are strongly discouraged, but technically valid. 
https://devmanual.gentoo.org/ebuild-writing/file-format/

File: app-crypt/zuluplay/metadata.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+		<email>fdsfgs@krutt.org</email>
+		<name>soredake</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<upstream>
+		<remote-id type="github">mhogomchungu/zuluCrypt</remote-id>
+	</upstream>
```
Comment (jonasstein) [inline app-crypt/zuluplay/metadata.xml]: please add longdescription

**Comment by soredake** [comment]
@jonasstein moved zuluCrypt -> zulucrypt and added long desc.

**Comment by soredake** [comment]
Problems with compiling zulucrypt with internal zuluplay is solved by https://github.com/mhogomchungu/zuluCrypt/commit/e33bc6ad24ec2a260cc68cb2f4c0dde3b9ebb421 and backported as patch to 5.3.0 version.

**Comment by soredake** [comment]
@dilfridge

**Comment by soredake** [comment]
Ping.

**Comment by soredake** [comment]
ping @jonasstein

**Comment by soredake** [comment]
ping @mgorny

**Comment by candrews** [comment]
Can you please add the `:=` slot operator to the cryptsetup dependency?

**Comment by titanofold** [comment]
Rebase, regenerate the manifest, and squash the PR, please.

The Manifest isn't valid anymore. We should only see BLAKE2B and SHA512.

**Comment by soredake** [comment]
@candrews @titanofold done

**Comment by candrews** [comment]
Thanks!

a64d121bad1eff5605d418572fff7bb85c63849c

**Pull request closed without merging**